src/Libraries/CoreNodeModels/Input/BasicInteractive.cs

```diff
@@ -44,13 +44,13 @@ public override NodeInputData InputData
             {
                 return new NodeInputData()
                 {
-                    Id = this.GUID.ToString("N"),
+                    Id = this.GUID,
```

Comment:
@QilongTang this leverages your changes to the GUID converter so it does get serialized with dashes.